tests/test_all_72_chat_combinations.py

```
@@ -0,0 +1,328 @@
+        return all_combinations
+
+    @pytest.mark.parametrize("combination_index", list(range(72)))
+    def test_chat_combination_by_index(self, client, csrf_token, combination_index):
+        """インデックスベースで全72組み合わせをテスト（APIキーローテーション使用）"""
+        combinations = self.get_all_72_combinations()
+        partner_type, situation, topic, model = combinations[combination_index]
+        
+        print(f"\n=== 組み合わせ {combination_index + 1}/72: {partner_type} × {situation} × {topic} × {model} ===")
+        
+        # APIキー管理システムの使用
+        manager = get_api_key_manager()
```

Ruff (0.12.2)

Local variable `manager` is assigned to but never used

Remove assignment to unused variable `manager`

(F841)